xmtp_mls/src/storage/encrypted_store/conversation_list.rs

```diff
@@ -53,17 +57,112 @@ pub struct ConversationListItem {
 }
 
 impl DbConnection {
-    pub fn fetch_conversation_list(&self) -> Result<Vec<ConversationListItem>, StorageError> {
-        let query = conversation_list
+    pub fn fetch_conversation_list<A: AsRef<GroupQueryArgs>>(
+        &self,
+        args: A,
+    ) -> Result<Vec<ConversationListItem>, StorageError> {
+        use crate::storage::schema::consent_records::dsl as consent_dsl;
+        use crate::storage::schema::conversation_list::dsl as conversation_list_dsl;
+
+        let GroupQueryArgs {
+            allowed_states,
+            created_after_ns,
+            created_before_ns,
+            limit,
+            conversation_type,
+            consent_state,
+            include_sync_groups,
+            include_duplicate_dms,
+        } = args.as_ref();
+        let mut query = conversation_list
             .select(conversation_list::all_columns())
+            .filter(conversation_list_dsl::conversation_type.ne(ConversationType::Sync))
             .into_boxed();
-        Ok(self.raw_query(|conn| query.load::<ConversationListItem>(conn))?)
+
+        //question: do we need to do this always? if yes, we can put it in the main view query instead of here!
```

Comment:
Do you mean is this flag always true?

Reply:
So, what you're saying I can keep it like this, right?

Reply:
If you want to always filter out duplicate DMs. This section of code will filter out duplicates.

Reply:
I will keep it like these to give the control to the sdk and the integrators.

Reply:
So just for clarity by default the duplicate dms are filtered out correct? Or will it be required to do it on my end?